workloads/kube-burner/common.sh

```diff
@@ -0,0 +1,124 @@
+
+# Check cluster's health
+if [[ ${CERBERUS_URL} ]]; then
```

Comment:
We might want to add a Cerberus check during/at the end of the run as well to make sure the upgraded cluster is healthy. We can query ripsaw as well, for example: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/network-perf/run_pod_network_test_fromgit.sh#L77. Think the benchmark will fail if Cerberus reports issues right? In that case we don't need an explicit check. Thoughts?

Reply:
Why do we need to check this, if the Operator does? seems unnecessary.

Reply:
@chaitanyaenr instead trusting in the curl,  I added the parameter cerberus_url to the CRD so now ripsaw will take care of checking cerberus status.